doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -520,7 +520,7 @@ Exercise 1: Language identification
 -----------------------------------
 
 - Write a text classification pipeline using a custom preprocessor and
-  ``CharNGramAnalyzer`` using data from Wikipedia articles as training set.
+  ``CountVectorizer`` set up to use character based n-grams, using data from Wikipedia articles as the training set.
```

Comment:
The solution to this exercise uses `TfidfVectorizer`:

https://github.com/scikit-learn/scikit-learn/blob/4e679023eb63fff9aa611ed7189fad7581b742a0/doc/tutorial/text_analytics/solutions/exercise_01_language_train_model.py#L31-L34

To keep the question consistent with the solution:

```suggestion
  ``TfidfVectorizer`` set up to use character based n-grams, using data from Wikipedia articles as the training set.
```